elections/steering/2024/README.md

```diff
@@ -117,31 +117,30 @@ Examples of contributions that would NOT be considered:
 
 <!-- While finalizing the dates in the schedule, ensure that:
 - The Steering Committee and candidate Q+A occurs at a public SC meeting
-  (usually a Monday).
+  (usually a Wednesday).
 - Deadline to submit voter exception forms and request a
   replacement ballot is ~3 days before voting closes.
 - Private announcement of results to SC members is at least ~2 days
   before private announcement to all candidates.
 - The interval between private announcement to all candidates and the
-  public announcement is 24-48 hours, ideally during a weekend.
+  public announcement is 24-48 hours.
```

Comment:
Note: I removed the "ideally during a weekend" stipulation due to the Steering meeting moving to Wednesdays. There is no reality where the interval between a Wednesday and a weekend is 24-48 hours.